test/lib/cql_test_env.cc

```diff
@@ -186,6 +186,7 @@ class single_node_cql_env : public cql_test_env {
 private:
     cql3::dialect test_dialect() {
         return cql3::dialect{
+            .duplicate_bind_variable_names_refer_to_same_variable = _db.local().get_config().cql_duplicate_bind_variable_names_refer_to_same_variable(),
```

Comment:
Looks like users have to make up their minds about which mode to use very early and choose one or the other. How does Cassandra solve this? Do they also have a config?

Reply:
After reading the comments above, it looks like this is feature flag that the user has to consciously update on a per-need basis. Is that correct?

Reply:
Cassandra only has duplicate_bind_variable_names_refer_to_same_variable = false. This is nonsensical if one uses named bind variables and uses the keyword API in the driver. However, it can work if one uses named bind variables but the positional API in the driver, which in fact was seen in the field.